console/my-account/how-to/accept-invitation-new-scw-account.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to activate a new account after being invited to an Organization
+  description: This page explains how to activate a new account after being invited to an Organization
+  h1: How to activate a new account after being invited to an Organization
+  paragraph: This page explains how to change the color scheme of the Scaleway console
+tags: color-scheme change color theme dark-mode change-color
+dates:
+  validation: 2023-06-05
+  posted: 2023-06-15
+categories:
+  - console
+---
+
+When a Scaleway user invites you to join their Organization you receive an invitation via email. If you do not yet have a Scaleway account upon receiving this email, you will not be able to effectively join the Organization.
+
+You can follow the steps below to activate your Scaleway account and join the Organization you were invited to.
+
+1. Click **Accept Invitation** in the email you received inviting you to join an existing Scaleway Organization.
+    <Message type="important">
+      When you click the link, you are redirected to the Scaleway console. Since you do not have a Scaleway account at this point, you will not be able to log into the console.
+    </Message>
+2. Go back to your personal email mailbox.
+3. Check if you have received an email from Scaleway with the following subject: **[Action Required] Confirm Your Scaleway Account**.
+4. Open the email if you have received it.
+    <Message type="important">
+      If you have not received the email, check that it has not been sent to SPAM. If it is not in your SPAM folder, wait a while longer to see if you will receive the email. You can also contact the person who invited you to re-send the invitation. If none of these options work, open a ticket and [contact our support team](https://console.scaleway.com/support/tickets).
```

Comment:
Will they be able to open a ticket if they don't have an account?

Reply:
yes, I checked!